src/components/Article.jsx

```diff
@@ -11,71 +12,48 @@ export default function Article() {
   const { article_id } = useParams();
 
   useEffect(() => {
-    getArticle(article_id).then((body) => {
-      setArticle(body);
-    });
+    getArticle(article_id).then((body) => setArticle(body));
   }, []);
 
-  let formattedDate = "";
-  if (article.created_at) {
-    formattedDate = article.created_at.slice(0, 10);
-  } else {
+  let date = "";
+  if (article.created_at) date = article.created_at.slice(0, 10);
+  else
     return (
       <>
         <Header />
         <p>Loading article...</p>
       </>
     );
-  }
 
   function handleLike() {
     setLikes(likes + 1);
-
-    patchLikes(article_id).then((article) => {
+    patchLikes(article_id).then(() =>
       setArticle((article) => ({
         ...article,
         votes: article.votes + 1,
```

Comment:
You shouldn't need to do this because you are already optimistically adding a vote and it should persist on refresh. What you do need to do is undo the optimistic increase in a catch block 🙂